### PR TITLE
adds assignment six for review

### PR DESCRIPTION
**Pull request by aman09v**
What does this change do?
adds assignment six for review

Any additional information for the reviewer to start
NA

How should this be manually tested?
We need window OS in which Rust is installed. then execute this rust program

Which Trello task and sub-task does this change relate to?
NA

Are there any changes pending?
No

Does any team have to be notified of changes in this feature?
No

Definition of Done:
- [x] Is there >90% unit test code coverage?
- [ ] Does this PR add new dependencies? If so, please list out the same.
- [ ]  Will this feature require a new piece of infrastructure to be implemented?
- [ ] Is there appropriate logging included?
- [x] Does the project compile ok?
- [x] Have Clippy violations been fixed?
- [x] Have Code is properly formatted?

### CODE REVIEW

File: Assignments/assignment-six/src/ip_enum.rs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#[derive(PartialEq, Eq, Debug)]
+pub enum IpClass {
+    ClassA(String),
+    ClassB(String),
+    ClassC(String),
+    ClassD(String),
+    None,
+}
+
+/// This function classifies ip addresses into classes.
```
Comment (ankit-mogha) [inline Assignments/assignment-six/src/ip_enum.rs:10]: use a good comment

File: Assignments/assignment-six/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+mod ip_enum;
+mod point_position;
+mod test;
+
+/// This function uses functions to remove warning
```
Comment (ankit-mogha) [inline Assignments/assignment-six/src/lib.rs:5]: use a good comment

File: Assignments/assignment-six/src/point_position.rs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+#[derive(PartialEq, Eq, Debug)]
+pub enum Position {
+    First(Coordinate, Coordinate),
+    Second(Coordinate, Coordinate),
+    Third(Coordinate, Coordinate),
+    Fourth(Coordinate, Coordinate),
+    Origin,
+}
+
+/// This function finds the quadrant in which a point lies.
```
Comment (ankit-mogha) [inline Assignments/assignment-six/src/point_position.rs:15]: use a good comment

File: Assignments/assignment-six/src/ip_enum.rs (hunk 1 above)
Comment (aman09v) [reply Assignments/assignment-six/src/ip_enum.rs]: done

File: Assignments/assignment-six/src/lib.rs (hunk 2 above)
Comment (aman09v) [reply Assignments/assignment-six/src/lib.rs:5]: done

File: Assignments/assignment-six/src/point_position.rs (hunk 3 above)
Comment (aman09v) [reply Assignments/assignment-six/src/point_position.rs]: done

**Review by ankit-mogha — APPROVED** [review]

**Review by Dheerajjnv — APPROVED** [review]

**Pull request merged**